packages/back-end/types/experimentLaunchChecklist.d.ts

```diff
@@ -3,6 +3,7 @@ export interface ChecklistTask {
   completionType: "manual" | "auto";
   url?: string;
   customFieldId?: string;
+  projects: string[];
```

Comment:
This field needs to be marked as optional to enforce type safety (or add a JIT migration)
```suggestion
  projects?: string[];
```

Reply:
Ah, good catch. TY